index.js

```diff
@@ -1276,6 +1276,25 @@
   }
   S.maybe = def('maybe', {}, [b, $.Function, $Maybe(a), b], maybe);
 
+  //# maybe_ :: (() -> b) -> (a -> b) -> Maybe a -> b
+  //.
+  //. Similiar to `maybe` but for use in cases where you want to compute the
+  //. Nothing case lazily. So, instead takes a value for the first argument,
+  //. it takes a function that will be called without arguments if the `Maybe`
+  //. is known to be `Nothing`.
```

Comment:
I think we could get away with a shorter description. How does this sound?

    Variant of [`maybe`](#maybe) which takes a thunk so the default value
    is only computed if required.

We should reference `maybe_` in the description of `maybe`:

    See also [`maybe_`](#maybe_).